content/docs/trust/trust-manager/README.md

```diff
@@ -131,7 +131,13 @@ JKS has been supported since v0.5.0, and PKCS#12 since v0.7.0.
 
 Applications consuming JKS and PKCS#12 trust stores often require a password to be set for legacy reasons. These passwords are often security theater - either they use very weak encryption or the passwords are provided in plaintext next to the files they encrypt which defeats the purpose of having them.
 
-Trust bundles do not contain private keys, and so for most use cases there wouldn't be any security benefit to encrypting them. As such, passwords for trust stores are hard-coded to `changeit` for JKS and `""` (the empty string or "password-less") for PKCS#12. Future releases of trust-manager may make these passwords configurable.
+Trust bundles do not contain private keys, and so for most use cases there wouldn't be any security benefit to encrypting them. As such, passwords for trust stores are set by default to `changeit` for JKS and `""` (the empty string or "password-less") for PKCS#12. 
+
+Recent releases will allow you to change that password by setting the bundle yaml file `spec.target.additionalFormats.jks.password` and `spec.target.additionalFormats.pkcs12.password`. 
+
+Older releases have the current default values hard-coded and they can not be changed. For more information read [why password are not helpful](https://cert-manager.io/docs/faq/#why-are-passwords-on-jks-or-pkcs12-files-not-helpful).
```

Comment:
Could you please make this link a relative link to the document mentioned?
I think it should be this 🤔 :

```markdown
[why password are not helpful](../../docs/faq/#why-are-passwords-on-jks-or-pkcs12-files-not-helpful)
```

Reply:
Done

Reply:
Sorry @eduyubero i just tried it on the preview and the one I thought worked is wrong. It takes you to: https://deploy-preview-1485--cert-manager-website.netlify.app/docs/docs/faq/#why-are-passwords-on-jks-or-pkcs12-files-not-helpful

Double `docs`.

So the intended path is.. https://deploy-preview-1485--cert-manager-website.netlify.app/docs/faq/#why-are-passwords-on-jks-or-pkcs12-files-not-helpful

I think the actual link should be "../../faq/#why-are-passwords-on-jks-or-pkcs12-files-not-helpful".

Goes back 2 directory which is `docs` and then into `faq`. Sorry It's difficult to check without checking out the project and opening in vscode or something :)

Reply:
Updated.